set.mm

```diff
@@ -633662,6 +633662,96 @@ deduction version (Contributed by metakunt, 22-May-2024.) $)
       $.
   $}
 
+  ${
+    $d A x $.  $d F x $.  $d G x $.  $d ph x $.
+    eqfnfv2d.1 $e |- ( ph -> F Fn A ) $.
```

Comment:
This is not a deduction version of eqfnfv2, since it proves the reverse direction of the bi-conditional only.  Use a different name (with suffix -2, for example), to allow for a later true deduction version.

Reply:
Name changed